### PR TITLE
Enhancing faas cli

### PR DESCRIPTION
**Pull request by trisberg**
- adding build and create commands

- aligning options with kubectl
  using -f for path/filename and -n for function name

- wrapping kubectl create as ./faas create

- updating README using the new commands

### CODE REVIEW

File: faas [hunk 1]
```diff
@@ -3,25 +3,34 @@
...
-[*] -f  | --function - the name of the function
-    -p  | --path - the path to the function code (defaults to current directory)
-    -v  | --version - the version of the function created (defaults to v0001)
+faas create -f <path>
+  This command will create the resource[s] based on the resource definition[s] that the path points to
 
-[*] = Required arguments
+Options:
+
+  -n  | --name - the name of the function (defaults to nake of current directory)
```
Comment (markfisher) [inline faas]: typo: make

File: faas [hunk 2]
```diff
@@ -3,25 +3,34 @@
...
+  specified for the image that is built
 
-[*] -f  | --function - the name of the function
-    -p  | --path - the path to the function code (defaults to current directory)
-    -v  | --version - the version of the function created (defaults to v0001)
+faas create -f <path>
+  This command will create the resource[s] based on the resource definition[s] that the path points to
 
-[*] = Required arguments
+Options:
```
Comment (markfisher) [inline faas]: shouldn't `-n` and `-v` only be supported when using `faas build`?

it seems like we need separate usage... the top level should just show sub-commands, and then for specific usage, we'd support something like `faas <command> -h`

Comment (trisberg) [reply faas]: Yeah, we could do that. Let me add a commit

**Comment by markfisher** [comment]
should we add `apply` in addition to `create`? I'm thinking that might be useful for topics since multiple functions are sharing them

**Comment by markfisher** [comment]
I'd also like to add log-tailing here as `faas logs -n function` (can simplify that part of the README significantly). I've been using a script for that locally. In addition to the function name, we'd need an option for choosing sidecar vs. main container. My script takes `-c sidecar` (default is `main`) but really it's just a binary choice given our model, so it could perhaps be simpler than that.

**Comment by markfisher** [comment]
rebased and merged as https://github.com/markfisher/sk8s/commit/a7a4797510c3707a0c48d26e35c6208125487c10
...along with these changes: https://github.com/markfisher/sk8s/commit/6a7745baad003770a162ca67a0e7dda8f9492759

**Pull request closed without merging**